src/plugins/intel_gpu/tests/test_cases/reorder_gpu_test.cpp

```diff
@@ -1602,7 +1602,7 @@ TEST(reorder_gpu_opt, mean_mul_val)
     cldnn::mem_lock<float> ptr(output, get_test_stream());
     float* a_ptr = answers;
     for (auto& val : ptr)
-        EXPECT_FLOAT_EQ(*(a_ptr++), val);;
+        EXPECT_FLOAT_EQ(*(a_ptr++), val);
```

Comment:
[random spot] Please add instances for Softmax-8 functional tests

Reply:
Softmax-8 functional tests already exist